common/utils/backgrounds.js

```diff
@@ -0,0 +1,3 @@
+export const transparentGif = 'https://prismic-io.s3.amazonaws.com/wellcomecollection%2F7ca32858-e347-4282-acaf-c55572961736_transparent.gif';
+export const repeatingLs = 'https://prismic-io.s3.amazonaws.com/wellcomecollection%2F805ad61b-fba6-4dc1-b2d3-55dbcda0c9f1_ls_svg.svg';
+export const repeatingLsBlack = 'https://prismic-io.s3.amazonaws.com/wellcomecollection%2Fe483a8f3-f7bb-4c3f-8081-8f5d8875230e_ls_svg_black.svg';
```

Comment:
Another 🥇

Reply:
Still don't know what it is! I'll assume it's still good.

<img width="160" alt="screen shot 2018-10-04 at 16 07 17" src="https://user-images.githubusercontent.com/1394592/46483463-9b373d80-c7ef-11e8-8425-0f5e58a5c365.png">